### PR TITLE
qase-pytest: update docs

### PR DESCRIPTION
**Pull request by NickVolynkin**
# qase-pytest: update docs

* Update code examples, remove the wrong info about attachments not uploaded.
* Put the example with creating a test run in Qase first, because it's the
  first option to try and the most popular one.
* Update the Configuration table: rewrite descriptions, update and explain default values.
* Start the v5 to v6 migration guide.
* Update grammar, style, and formatting in multiple places.

# qase-pytest-commons: complete runs by default

### CODE REVIEW

File: qase-pytest/README.md [hunk 1]
```diff
@@ -1,57 +1,60 @@
...
+			"title": "Test run title"
 		},
-        "plan": {
-            "id": 1
-        },
-		"defect": true,
-		"project": "YOUR_PROJECT_CODE",
-		"chunk": 200
+		"batch": {
+          "size": 100
```
Comment (gibiw) [inline qase-pytest/README.md:47]: problem with format
image

File: qase-pytest/docs/CONFIGURATION.md [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
+| Description of the Qase test run               | `testops.run.description`  | `QASE_TESTOPS_RUN_DESCRIPTION`  | `--qase-testops-run-description`  | None, leave empty                       | No       | Any string                 |
+| Create test run using a test plan              | `testops.plan.id`          | `QASE_TESTOPS_PLAN_ID`          | `--qase-testops-plan-id`          | None, don't use plans for the test run  | No       | Any integer                |
+| Complete test run after running tests          | `testops.run.complete`     | `QASE_TESTOPS_RUN_COMPLETE`     | `--qase-testops-run-complete`     | `True`                                  | No       | `true`, `false`            |
+| ID of the Qase test run to report results      | `testops.run.id`           | `QASE_TESTOPS_RUN_ID`           | `--qase-testops-run-id`           | None, create a new test run             | No       | Any integer                |
+| Batch size for uploading test results          | `testops.batch.size`       | `QASE_TESTOPS_BATCH_SIZE`       | `--qase-testops-batch-size`       | `200s`                                  | No       | 1 to 2000                  |
```
Comment (gibiw) [inline qase-pytest/docs/CONFIGURATION.md:29]: why 200**s**?

**Review by gibiw — APPROVED** [review]

File: qase-pytest/docs/CONFIGURATION.md (hunk 2 above)
Comment (NickVolynkin) [reply qase-pytest/docs/CONFIGURATION.md]: fixed

File: qase-pytest/README.md (hunk 1 above)
Comment (NickVolynkin) [reply qase-pytest/README.md]: fixed

**Pull request merged**